### PR TITLE
Add introduction for Guthrie Tabios

### PR DESCRIPTION
**Pull request by tabiosg**
Add my introduction to the main branch.

### CODE REVIEW

**Review by jfwoods — APPROVED** [review]

File: introductions/GuthrieTabios.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name: Guthrie Tabios
+Pronouns: he/him/his
+Year: Junior
+Major: Computer Engineering
+
+This is my third year being on the rover embedded software team!
+I learned C++ from university classes and learned C and Python from working on projects for rover.
+
+In my free time, I play League of Legends and watch anime =)
```
Comment (WinsonChen114) [inline introductions/GuthrieTabios.txt]: Add a period to the end of the sentence

File: introductions/GuthrieTabios.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name: Guthrie Tabios
+Pronouns: he/him/his
+Year: Junior
+Major: Computer Engineering
+
+This is my third year being on the rover embedded software team!
```
Comment (WinsonChen114) [inline introductions/GuthrieTabios.txt]: Mrover*

File: introductions/GuthrieTabios.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+I learned C++ from university classes and learned C and Python from working on projects for rover.
```
Comment (WinsonChen114) [inline introductions/GuthrieTabios.txt]: Mrover*

**Review by WinsonChen114 — CHANGES_REQUESTED** [review]
See comments

File: introductions/GuthrieTabios.txt (hunk 3 above)
Comment (tabiosg) [reply introductions/GuthrieTabios.txt]: MRover*

File: introductions/GuthrieTabios.txt (hunk 1 above)
Comment (tabiosg) [reply introductions/GuthrieTabios.txt]: Thank you Winson!

File: introductions/GuthrieTabios.txt (hunk 2 above)
Comment (tabiosg) [reply introductions/GuthrieTabios.txt]: Thank you Winson Chen!

File: introductions/GuthrieTabios.txt (hunk 3 above)
Comment (tabiosg) [reply introductions/GuthrieTabios.txt]: Thank you WinsonChen114!

Comment (WinsonChen114) [reply introductions/GuthrieTabios.txt]: You're welcome!

**Review by WinsonChen114 — APPROVED** [review]
Looks good to me!

**Pull request merged**